_src/pattern-library/sass/components/_forms.scss

```diff
@@ -0,0 +1,455 @@
+// #INPUT TEXT
+// ----------------------------
+.input-text {
+    width: input-width(base);
+
+    &:focus,
+    &.is-active {
+        border-color: $input-border-default-focus;
+        color: $input-text-default-hover;
+    }
+
+    &:disabled,
+    &.is-disabled {
+        border-color: $input-border-disabled;
+        background: $input-background-disabled;
+    }
+
+    &.input-simple {
+        padding: spacing-verical(small) 0;
+        border-width: 0 0 2px 0;
+
+        &:focus,
+        &.is-active {
+            border-color: $input-simple-border-focus;
+            background: $input-simple-background-focus;
+        }
+    }
+}
+
+// ----------------------------
+// #SELECT
+// ----------------------------
+.wrapper-replace-select {
+    position: relative;
+    margin-bottom: spacing-vertical(small); // although this is on the wrapper, since we're positioning absolutley the replaced UI, we need to add a bottom margin here too
+}
+
+.input-select {
+    width: input-width(base);
+    height: rem($baseline*2);
+    line-height: spacing-vertical(small);
+
+    &:focus,
+    &.is-active {
+        border-color: $input-border-default-focus;
+        color: $input-text-default-hover;
+    }
+
+    &:disabled,
+    &.is-disabled {
+        border-color: $input-border-disabled;
+        background: $input-background-disabled;
+    }
+
+    &.is-replaced {
+        position: relative;
+        top: rem($baseline/4);
+        left: 0;
+        z-index: z-index(mid-front);
+        opacity: 0;
+    }
+}
+
+// custom select styling
+// this is the replacement select menu
+.wrapper-custom-select {
+    @extend .input;
+    display: inline;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: input-width(base);
+    line-height: 100%; // centers the text vertically while matching height of replacement menu
+    z-index: z-index(base);
+
+    svg {
+        @include size(rem($baseline));
+        position: absolute;
+        right: rem(($baseline/2));
+        fill: $input-border-default;
+    }
+
+    &:focus,
+    &.is-active {
+        border: 1px solid $input-border-default-focus;
+
+        svg {
+            @include transition(all timing(xx-fast) ease-in-out 0s);
+            fill: $input-border-default-focus;
+        }
+    }
+
+    &:disabled,
+    &.is-disabled {
+        color: $input-text-disabled;
+        cursor: not-allowed;
+    }
+}
+
+// ----------------------------
+// #RADIO / CHECKBOX
+// ----------------------------
+.wrapper-replace-radio,
+.wrapper-replace-checkbox {
+    display: inline-block;
+    position: relative;
+    width: 1rem;
+}
+
+.wrapper-custom-radio,
+.wrapper-custom-checkbox {
+    display: inline-block;
+    z-index: z-index(base);
+
+    &.is-selected {
+
+    }
+}
+
+.input-radio,
+.input-checkbox {
+
+    &:disabled,
+    &.is-disabled {
+
+    }
+
+    &.replace-checkbox,
+    &.replace-radio {
+        @extend %a11y-hide;
+
+        & + .input-label:before {
```

Comment:
Nope. This is what gives the label the bold style on selected checkboxes or radio buttons. This targets the next sibling rather than the child.